tests/targetAcquisition_geolocation_test.py

```diff
@@ -65,4 +65,4 @@ def test_targetAcquisition_to_geolocation(get_image):
     assert coordinates_and_telemetry != None
 
     assert check2 == True
-    assert geo_coordinates != None
```

Comment:
Are these tests all stable?

Reply:
I think on my laptop they are stable.
The reason that we still include tensorflow is that they are still used in our code. 
I installed them for it to pass the test